### PR TITLE
Added Microsoft 365 Defender advanced hunting queries

### PR DESCRIPTION
**Pull request by vcartman**
Thank you for your contribution! Some pointers to get it merged quickly:

For contributions in `Hunting/`:
Added Markdown file for "Microsoft 365 Defender advanced hunting queries" and updated the README.md with the link to the Log4-Microsoft-Defender.md file in the generic Hunting readme.md file

### CODE REVIEW

**Comment by vcartman** [comment]
Added for Fortinet Fix and Mitigation information where possible.

File: software/README.md [hunk 1]
```diff
@@ -847,9 +847,9 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
 |Fortinet| FortiOS (includes FortiGate & FortiWiFi)| |Not Vuln||[source](https://www.fortiguard.com/psirt/FG-IR-21-245)|
 |Fortinet| FortiPhish Cloud| |Not Vuln||[source](https://www.fortiguard.com/psirt/FG-IR-21-245)|
 |Fortinet| FortiPolicy| |Vulnerable||[source](https://www.fortiguard.com/psirt/FG-IR-21-245)|
-|Fortinet| FortiPortal | |Vulnerable||[source](https://www.fortiguard.com/psirt/FG-IR-21-245)|
+|Fortinet| FortiPortal | Fix |Vulnerable| Fixed in version 6.0.8 and 5.3.8 |[source](https://www.fortiguard.com/psirt/FG-IR-21-245)|
```
Comment (maertsen) [inline software/README.md:850]: Please use the version field for the version and the status field for the status.

File: software/README.md [hunk 2]
```diff
@@ -847,9 +847,9 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
...
 |Fortinet| FortiRecorder| |Not Vuln||[source](https://www.fortiguard.com/psirt/FG-IR-21-245)|
-|Fortinet| FortiSIEM| |Vulnerable||[source](https://www.fortiguard.com/psirt/FG-IR-21-245)|
+|Fortinet| FortiSIEM| Workaround |Vulnerable| [Mitigation steps](https://community.fortinet.com/t5/FortiSIEM/Techincal-Tip-FortiSIEM-Log4j-Mitigating-CVE-2021-44228/ta-p/201008) |[source](https://www.fortiguard.com/psirt/FG-IR-21-245)|
```
Comment (maertsen) [inline software/README.md:852]: same, duplicate status

File: software/README.md [hunk 3]
```diff
@@ -859,7 +859,7 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
 |Fortinet| ShieldX| |Vulnerable||[source](https://www.fortiguard.com/psirt/FG-IR-21-245)|
 |F-Secure| Endpoint Proxy | 13-15| Fix| |[source](https://status.f-secure.com/incidents/sk8vmr0h34pd)|
 |F-Secure| Policy Manager | 13-15| Fix| |[source](https://status.f-secure.com/incidents/sk8vmr0h34pd)|
-|F-Secure| Policy Manager Proxy| 13-15| Fix| |[source](https://status.f-secure.com/incidents/sk8vmr0h34pd)|
+|F-Secure| Policy Manager Proxy| 13-15| Fix | |[source](https://status.f-secure.com/incidents/sk8vmr0h34pd)|
```
Comment (maertsen) [inline software/README.md:862]: unrelated change, please remove

File: software/README.md [hunk 4]
```diff
@@ -827,7 +827,7 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
 |Forcepoint |Web Security                                                                         ||Investigation | |[source](https://support.forcepoint.com/s/article/Apache-log4j-Zero-Day-RCE-Vulnerability-CVE-2021-44228)|
 |Forescout | | | Investigation | | [source](https://www.forescout.com/blog/forescout%e2%80%99s-response-to-cve-2021-44228-apache-log4j-2/)
 |ForgeRock        | Autonomous Identity |  | Workaround | all other ForgeRock products not vuln | [source](https://backstage.forgerock.com/knowledge/kb/book/b21824339#1_bzBa) |
-|Fortinet| FortiAIOps| |Vulnerable||[source](https://www.fortiguard.com/psirt/FG-IR-21-245)|
+|Fortinet| FortiAIOps| version < 1.0.1 |Vulnerable| Fixed in version 1.0.2 |[source](https://www.fortiguard.com/psirt/FG-IR-21-245)|
```
Comment (maertsen) [inline software/README.md:830]: Can you make this `| 1.0.2 | Fixed |`?

**Review by maertsen — CHANGES_REQUESTED** [review]
hi @vcartman, thanks for your contribution. Some review comments on the software bits.

**Review by dionvankooten — APPROVED** [review]

**Review by dionvankooten — APPROVED** [review]

**Pull request merged**